--- lua/user/bufferline.lua.orig	2023-06-16 20:28:52.000000000 -0700
+++ lua/user/bufferline.lua	2023-06-16 20:29:09.000000000 -0700
@@ -3,7 +3,7 @@
   return
 end
 
-bufferline.setup {
+bufferline.setup({
   options = {
     numbers = "none", -- | "ordinal" | "buffer_id" | "both" | function({ ordinal, id, lower, raise }): string,
     close_command = "Bdelete! %d", -- can be a string | function, see "Mouse actions"
@@ -13,8 +13,8 @@
     -- NOTE: this plugin is designed with this icon in mind,
     -- and so changing this is NOT recommended, this is intended
     -- as an escape hatch for people who cannot bear it for whatever reason
-	indicator_icon = nil,
-    indicator = { style = "icon", icon = "▎"},
+    indicator_icon = nil,
+    indicator = { style = "icon", icon = "▎" },
     buffer_close_icon = "",
     -- buffer_close_icon = '',
     modified_icon = "●",
@@ -74,7 +74,7 @@
   },
   highlights = {
     fill = {
-      fg = { attribute = "fg", highlight = "#ff0000" },
+      fg = { attribute = "fg", highlight = "Tabline" },
       bg = { attribute = "bg", highlight = "TabLine" },
     },
     background = {
@@ -165,4 +165,4 @@
       bg = { attribute = "bg", highlight = "Normal" },
     },
   },
-}
+})
--- minimal/minimal.lua.orig	2023-06-16 20:36:39.000000000 -0700
+++ minimal/minimal.lua	2023-06-16 20:41:51.000000000 -0700
@@ -6,12 +6,12 @@
 -- vim.opt.runtimepath:prepend "PATH/neovim/nvim-lspconfig"
 -- vim.opt.runtimepath:prepend "PATH/williamboman/mason.nvim"
 -- vim.opt.runtimepath:prepend "PATH/williamboman/mason-lspconfig.nvim"
-vim.opt.runtimepath:prepend "/home/benjamin/.local/share/nvim-LaTeX/site/pack/packer/start/cmp-omni"
-vim.opt.runtimepath:append  "/home/benjamin/.local/share/nvim-LaTeX/site/pack/packer/start/cmp-omni/after"
-vim.opt.runtimepath:prepend "/home/benjamin/.local/share/nvim-LaTeX/site/pack/packer/start/nvim-cmp"
-vim.opt.runtimepath:append  "/home/benjamin/.local/share/nvim-LaTeX/site/pack/packer/start/nvim-cmp/after"
-vim.opt.runtimepath:prepend "/home/benjamin/.local/share/nvim-LaTeX/site/pack/packer/start/vimtex"
-vim.opt.runtimepath:append  "/home/benjamin/.local/share/nvim-LaTeX/site/pack/packer/start/vimtex/after"
+vim.opt.runtimepath:prepend vim.fn.stdpath("data") .. "/site/pack/packer/start/cmp-omni"
+vim.opt.runtimepath:append  vim.fn.stdpath("data") .. "/site/pack/packer/start/cmp-omni/after"
+vim.opt.runtimepath:prepend vim.fn.stdpath("data") .. "/site/pack/packer/start/nvim-cmp"
+vim.opt.runtimepath:append  vim.fn.stdpath("data") .. "/site/pack/packer/start/nvim-cmp/after"
+vim.opt.runtimepath:prepend vim.fn.stdpath("data") .. "/site/pack/packer/start/vimtex"
+vim.opt.runtimepath:append  vim.fn.stdpath("data") .. "/site/pack/packer/start/vimtex/after"
 
 vim.cmd[[filetype plugin indent on]]
 
