--- lua/core/settings.lua.orig	2023-06-05 18:26:45.635868085 -0700
+++ lua/core/settings.lua	2023-06-05 18:25:03.972025817 -0700
@@ -79,7 +79,7 @@
 -- Tools that should be installed by Mason
 M.tools = {
   -- Formatter
-  "black",
+  -- "black",
   "prettier",
   "stylua",
   "shfmt",
@@ -88,7 +88,7 @@
   "shellcheck",
   "tflint",
   "yamllint",
-  "ruff",
+  -- "ruff",
   -- DAP
   "debugpy",
   "codelldb",
