--- lua/user/plugins/enabled.lua.orig	2023-07-26 18:08:47.408693395 -0700
+++ lua/user/plugins/enabled.lua	2023-07-26 18:10:13.709742780 -0700
@@ -1,6 +1 @@
-return {
-  {
-    "wakatime/vim-wakatime",
-    enabled = true,
-  },
-}
+return {}
--- lua/user/plugins/disabled.lua.orig	2023-07-26 18:09:30.201210869 -0700
+++ lua/user/plugins/disabled.lua	2023-07-26 18:10:02.369603609 -0700
@@ -3,4 +3,8 @@
     "monaqa/dial.nvim",
     enabled = false,
   },
+  {
+    "wakatime/vim-wakatime",
+    enabled = false,
+  },
 }
