--- lua/user/plugins.lua.00	2023-08-31 07:27:40.645716296 -0700
+++ lua/user/plugins.lua	2023-08-31 07:26:34.984834218 -0700
@@ -123,9 +123,9 @@
   },
 
   -- AI
-  { 'zbirenbaum/copilot.lua', conf = 'copilot' },
-  { 'dpayne/CodeGPT.nvim', conf = 'codegpt' },
-  { 'piersolenski/wtf.nvim', conf = 'wtf' },
+  -- { 'zbirenbaum/copilot.lua', conf = 'copilot' },
+  -- { 'dpayne/CodeGPT.nvim', conf = 'codegpt' },
+  -- { 'piersolenski/wtf.nvim', conf = 'wtf' },
 
   -- Quickfix/Loclist
   { 'kevinhwang91/nvim-bqf', conf = 'bqf', ft = 'qf', event = 'QuickFixCmdPre' },
--- lua/user/statusline/init.lua.00	2023-08-31 07:28:57.962748471 -0700
+++ lua/user/statusline/init.lua	2023-08-31 07:30:22.771873770 -0700
@@ -7,8 +7,8 @@
 
 require 'user.statusline.lsp'
 require 'user.statusline.dap'
-require 'user.statusline.codegpt'
-require 'user.statusline.wtf'
+-- require 'user.statusline.codegpt'
+-- require 'user.statusline.wtf'
 
 local fn = require 'user.fn'
 
@@ -188,14 +188,14 @@
 }
 
 config.components.active[2] = {
-  {
-    provider = 'codegpt',
-    hl = { fg = 'mistyrose', bold = false },
-  },
-  {
-    provider = 'wtf',
-    hl = { fg = 'mistyrose', bold = false },
-  },
+  -- {
+  --   provider = 'codegpt',
+  --   hl = { fg = 'mistyrose', bold = false },
+  -- },
+  -- {
+  --   provider = 'wtf',
+  --   hl = { fg = 'mistyrose', bold = false },
+  -- },
   {
     provider = 'lsp_progress',
     hl = { fg = 'mistyrose', bold = false },
--- lua/user/settings.lua.00	2023-08-31 07:45:08.275384913 -0700
+++ lua/user/settings.lua	2023-08-31 07:45:41.347810417 -0700
@@ -153,7 +153,8 @@
   end
 end
 
-setup_provider('python3', { vim.env.HOME, '/.asdf/shims/python3' })
+-- setup_provider('python3', { vim.env.HOME, '/.asdf/shims/python3' })
+vim.g.python3_host_prog = vim.fn.exepath("python3")
 setup_provider('node', { vim.env.XDG_DATA_HOME, '/yarn/global/node_modules/neovim/bin/cli.js' })
 setup_provider('ruby', { '/usr/bin/ruby' })
 
--- lua/user/mappings.lua.00	2023-08-31 07:49:36.746835060 -0700
+++ lua/user/mappings.lua	2023-08-31 07:51:00.895914918 -0700
@@ -1317,8 +1317,8 @@
 m.tnoremap(xk [[<C-/>]], toggleterm_smart_toggle, "ToggleTerm: Smart Toggle")
 
 ---- dpayne/CodeGPT.nvim
-m.nnoremap(xk [[<M-c>]], [[<cmd>FineCmdline Chat <Cr>]], "CodeGPT: Chat")
-m.vnoremap(xk [[<M-c>]], [[<cmd>FineCmdline '<,'>Chat <Cr>]], "CodeGPT: Chat")
+-- m.nnoremap(xk [[<M-c>]], [[<cmd>FineCmdline Chat <Cr>]], "CodeGPT: Chat")
+-- m.vnoremap(xk [[<M-c>]], [[<cmd>FineCmdline '<,'>Chat <Cr>]], "CodeGPT: Chat")
 
 ---- romgrk/nvim-treesitter-context
 m.nnoremap([[<leader>tsc]], [[<cmd>TSContextToggle<Cr>]], "Treesitter Context: Toggle")
