--- lua/core/utils/functions.lua.00	2023-09-18 14:36:55.820447915 -0700
+++ lua/core/utils/functions.lua	2023-09-18 14:42:12.912381006 -0700
@@ -170,11 +170,11 @@
 ---load user config file .nvim_config.lua
 ---@return table
 M.load_user_config = function()
-  local home = os.getenv("XDG_CONFIG_HOME")
-    or os.getenv("HOME")
-    or os.getenv("USERPROFILE")
-    or (os.getenv("HOMEDRIVE") .. os.getenv("HOMEPATH"))
-  local config_file = home .. M.path_separator() .. ".nvim_config.lua"
+  -- local home = os.getenv("XDG_CONFIG_HOME")
+  --   or os.getenv("HOME")
+  --   or os.getenv("USERPROFILE")
+  --   or (os.getenv("HOMEDRIVE") .. os.getenv("HOMEPATH"))
+  local config_file = vim.fn.stdpath("config") .. M.path_separator() .. ".nvim_config.lua"
   local ok, err = pcall(dofile, config_file)
   if not ok then
     M.notify("Can not load user config: " .. err, vim.log.levels.INFO, "core.utils")
--- .nvim_config.lua.00	2023-09-18 14:42:56.244920870 -0700
+++ .nvim_config.lua	2023-09-18 14:42:49.372835223 -0700
@@ -0,0 +1,15 @@
+-- see ./lua/core/config/defaults.lua for all
+-- possible values and a short description
+return {
+  options = {
+    relativenumber = true,
+  },
+  plugins = {
+    alpha = {
+      disable_dashboard_header = true,
+    },
+    spectre = {
+      enable = false,
+    },
+  },
+}
