--- lua/theprimeagen/init.lua.orig	2023-07-26 08:09:05.467618116 -0700
+++ lua/theprimeagen/init.lua	2023-07-26 08:16:01.292438215 -0700
@@ -1,8 +1,9 @@
 require("theprimeagen.set")
 require("theprimeagen.remap")
+require("theprimeagen.packer")

 -- DO NOT INCLUDE THIS
-vim.opt.rtp:append("~/personal/streamer-tools")
+-- vim.opt.rtp:append("~/personal/streamer-tools")
 -- DO NOT INCLUDE THIS

 local augroup = vim.api.nvim_create_augroup
--- lua/theprimeagen/packer.lua.orig	2023-07-26 08:03:52.348071885 -0700
+++ lua/theprimeagen/packer.lua	2023-07-26 08:06:14.249666421 -0700
@@ -1,7 +1,27 @@
--- This file can be loaded by calling `lua require('plugins')` from your init.vim
+local fn = vim.fn
+-- Automatically install packer
+local install_path = fn.stdpath("data") .. "/site/pack/packer/start/packer.nvim"
+if fn.empty(fn.glob(install_path)) > 0 then
+	PACKER_BOOTSTRAP = fn.system({
+		"git",
+		"clone",
+		"--depth",
+		"1",
+		"https://github.com/wbthomason/packer.nvim",
+		install_path,
+	})
+	print("Installing packer close and reopen Neovim...")
+	vim.cmd([[packadd packer.nvim]])
+end
+
+-- Use a protected call so we don't error out on first use
+local status_ok, packer = pcall(require, "packer")
+if not status_ok then
+  return
+end

 -- Only required if you have packer configured as `opt`
-vim.cmd.packadd('packer.nvim')
+-- vim.cmd.packadd('packer.nvim')

 return require('packer').startup(function(use)
   -- Packer can manage itself
--- lua/theprimeagen/remap.lua.orig	2023-07-26 08:03:40.451939373 -0700
+++ lua/theprimeagen/remap.lua	2023-07-26 08:06:56.862149657 -0700
@@ -42,7 +42,7 @@
 vim.keymap.set("n", "<leader>s", [[:%s/\<<C-r><C-w>\>/<C-r><C-w>/gI<Left><Left><Left>]])
 vim.keymap.set("n", "<leader>x", "<cmd>!chmod +x %<CR>", { silent = true })

-vim.keymap.set("n", "<leader>vpp", "<cmd>e ~/.dotfiles/nvim-Primeagen/.config/nvim-Primeagen/lua/theprimeagen/packer.lua<CR>");
+vim.keymap.set("n", "<leader>vpp", "<cmd>e ~/.config/nvim-Primeagen/lua/theprimeagen/packer.lua<CR>");
 vim.keymap.set("n", "<leader>mr", "<cmd>CellularAutomaton make_it_rain<CR>");

 vim.keymap.set("n", "<leader><leader>", function()
